### PR TITLE
[project] rubocop integration

### PR DESCRIPTION
**Pull request by 0exp**
- plus some gems definition redesign

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
+# frozen_string_literal: true
+
 source 'https://rubygems.org'
-gemspec
 
-group :development, :test do
-  gem 'surrealist'
-  gem 'benchmark-ips'
-end
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
```
Comment (akxcv) [inline Gemfile:5]: Don't think that's required.

File: Gemfile [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+
+# Specify your gem's dependencies in qonfig.gemspec
```
Comment (akxcv) [inline Gemfile]: Wrong gemspec :)

**Review by akxcv — COMMENTED** [review]
Thank you very much! Could you please correct a couple of small problems?

File: Gemfile (hunk 1 above)
Comment (0exp) [reply Gemfile:5]: default github mapping

File: Gemfile (hunk 2 above)
Comment (0exp) [reply Gemfile]: lel :)

File: Gemfile (hunk 1 above)
Comment (akxcv) [reply Gemfile:5]: yeah I know

**Review by akxcv — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (0exp) [reply Gemfile:5]: usefull and rquired ;)

**Pull request merged**